project/config/cvocni/config/unity_internal_link_checker.linksettings.yml

```diff
@@ -0,0 +1,2 @@
+site_url_list: "https://www.cvocni.org/\r\nhttps://cvocni.org/\r\nhttp://www.cvocni.org/\r\nhttp://cvocni.org/"
```

Comment:
We don't need these URL's anymore as there is background code to add them all in. I need to update the new site documentation, sorry.